docs/v2/guides/gutenberg.md

```diff
@@ -11,112 +11,123 @@ Timber works with the Block Editor (also called Gutenberg) out of the box. If yo
+    "style": [ "file:./my-block.css" ], // https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#category,
+    
+    "category": "formatting", // https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#category
+    "icon": "format-aside", // https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#icon
+    "keywords": ["my", "block"], // https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#keywords
+    "acf": { 
+        "mode": "preview", // https://www.advancedcustomfields.com/resources/acf-blocks-key-concepts/#acf-blocks-and-blockjson
+        "renderCallback": "my_acf_block_render_callback" // the function that will render the block we'll add later on
+    } 
 }
 ```
+Important note on the name property: You are not required to use acf as the namespace in the naming convention `namespace/block-name`, but you still need a value. The block-name will need to match your template name for when we render the block later on. 
 
-You create an extra array called `$context` with three values:
-- **block** - with all data like block title, alignment etc
-- **fields** - all custom fields - also all the fields created in **ACF**
-- **is_preview** - returns true during AJAX preview
+#### Register the Block
 
-Finally, you can create the template **block/example-block.twig**:
+Your site will need to know the block exists on init so we will employ the [register_block_type](https://developer.wordpress.org/reference/functions/register_block_type/) function witin our functions.php file and trigger the function on init. 
 
-```twig
-{#
-/**
- * Block Name: Example block
- *
- * This is the template that displays the example block.
- */
-#}
-
-{% if is_preview %}
-    <p>I will only appear in the editor.</p>
-{% endif %}
-
-<div id="example-{{ block.id }}" class="wrapper">
-    <h1>{{ fields.title }}</h1>
-    <p>{{ fields.description }}</p>
-</div>
-<style type="text/css">
-    #testimonial-{{ block.id }} {
-        background: {{ fields.background_color }};
-        color: {{ fields.text_color }};
-    }
-</style>
+The straight forward way to do this is posted below: 
+
+```php
+function register_acf_blocks() {
+    register_block_type( __DIR__ . '/blocks/my-block' ); //register_block_type will look in the current directory and register the block you specify
+    //add additional register_block_type functions for each of your custom blocks
+}
+add_action( 'init', 'register_acf_blocks' ); //trigger the register function on init
 ```
 
-If you would like to use an external stylesheet both inside of the block editor and the frontend you should add:
+Alternatively, you can dynamically register each block. This will keep our code DRY and prevent you from having to register each inidivual block every time you create a new one. 
```

Comment:
```suggestion
Alternatively, you can dynamically register each block. This will keep our code DRY and prevent you from having to register each individual block every time you create a new one.
```